### ISSUE TITLE
0 is not handled properly when used as variable value

### ISSUE THREAD

**Issue by Taiwar** [dittowords/ditto-react#25]
**Example code**
```js
// ditto/index.js
'test.id': {
    name: 'Test',
    text: 'Upload {{Amount}} selected',
    status: 'NONE',
    folder: null,
    variables: {
        Amount: {
            example: 1
        }
    }
}
```

```typescript
<Ditto
    componentId="test.id"
    variables={{ Amount: 0 }}
/>
```

**Result**
"Upload {{Amount}} selected"

**Expected result**
"Upload 0 selected"

The same is true for the value `""`.
Seems like there might be code using a falseness/falsey check on the value to determine whether to replace a variable or not.

**Comment by azjgard** [comment]
Hi @Taiwar, thanks so much for reporting! Sorry you're running into this issue. I'll look into it shortly.

**Comment by azjgard** [comment]
Hi @Taiwar, we just released [version 1.2.1](https://github.com/dittowords/ditto-react/releases/tag/v1.2.1) to resolve this issue.

Can you upgrade to the latest version and confirm that this resolves the reported problem?

**Comment by Taiwar** [comment]
Yes, problem is resolved after updating to 1.2.1. Thanks!
